app/src/main/java/com/extole/androidsdk/MainActivity.kt

```diff
@@ -1,48 +1,41 @@
 package com.extole.androidsdk
 
-import android.content.Context
-import android.content.Intent
 import android.os.Bundle
 import android.widget.Button
-import android.widget.EditText
 import android.widget.TextView
 import androidx.appcompat.app.AppCompatActivity
 import com.extole.android.sdk.Extole
+import dagger.hilt.android.AndroidEntryPoint
 import kotlinx.coroutines.GlobalScope
 import kotlinx.coroutines.launch
+import javax.inject.Inject
 
+@AndroidEntryPoint
 class MainActivity : AppCompatActivity() {
-    lateinit var sdkActivityContext: Context
+
+    @Inject
     lateinit var extole: Extole
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
-        sdkActivityContext = this
 
         GlobalScope.launch {
-            extole = ServiceLocator.getExtole(this@MainActivity)
-            val (zone, campaign) = extole.getZone("apply_for_card")
+            val (zone, campaign) = extole.fetchZone("cta_prefetch")
             runOnUiThread {
-                findViewById<EditText>(R.id.email_message).setText(
-                    zone.get("sharing.email.message").toString(), TextView.BufferType.NORMAL
-                )
-
-                findViewById<TextView>(R.id.copyright).setText(
-                    zone.get("sharing.facebook.description").toString(),
-                    TextView.BufferType.NORMAL
+                findViewById<Button>(R.id.menu_item).setText(
+                    zone.get("cta.title").toString(), TextView.BufferType.NORMAL
```

Comment:
"cta.title" or "title"?

Reply:
same below

Reply:
@apaniciuc let's change the zone: https://mobile-monitor.extole.io/zone/cta_prefetch response to:

```
{
   "title":"CTA Item",
   "image":"https://origin.xtlo.net/type=creativeArchive:clientId=1106500674:creativeArchiveId=7153594407382986000:version=1:coreAssetsVersion=66/images/generic-social.jpg",
   "touch_event":"cta_tap"
}
```

Reply:
I have changed this to
```
var mobileCta = {
    title: staticVariables.titleCopy,
    image: staticVariables.socialShareImageUrl,
    touch_event: "cta_tap"
};
```